doc/user/content/sql/functions/now_and_mz_logical_timestamp.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "now and mz_logical_timestamp functions"
+description: "In Materialize mz_logical_timestamp() acts the way now() does in most systems."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+In Materialize, `now()` doesn't represent the system time; it represents the time with timezone when the query was executed. By contrast, `mz_logical_timestamp()` represents the logical time at which the query executes.
+
+In most Materialize queries, you should use `mz_logical_timestamp()` where you might use `now()` in other systems, for example, when defining [temporal filters](https://materialize.com/temporal-filters/) or when creating materialized views from multiple sources with potentially conflicting system times.
+
+## Example
+
+For this example, you'll need to create a sample data source and create a materialized view from it for later reference.
+
+```sql
+--Create table
+CREATE TABLE events (
+    content text,
+    insert_ts numeric,
+    delete_ts numeric
+);
+--Create materialized view
+CREATE MATERIALIZED VIEW valid AS
+SELECT content, insert_ts, delete_ts
+FROM events
+WHERE mz_logical_timestamp() >= insert_ts
+  AND mz_logical_timestamp()  < delete_ts;
+```
+
+Next, you'll populate the table with timestamp data.
+
+```sql
+insert into events VALUES (
```

Comment:
nit: if we're capitalizing the CREATEs above, should we also capitalize INSERT INTO here (to keep it consistent)?